src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  &__icon {
+    width: 40px;
+    height: 40px;
+    img {
+      display: block;
+    }
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    padding: 0;
+    margin: 0;
+  }
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  &__link.is-active::after {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: '';
+    background-color: #00acdc;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+  }
```

Comment:
The `&__link.is-active::after` class is used to create a pseudo-element that represents the final state of the `&__link.is-active` element. However, the `position: absolute;`, `bottom: 0;`, `left: 0;`, `width: 100%;`, and `height: 4px;` properties are not necessary because they do not affect the appearance of the pseudo-element. Remove these properties.